srv/modules/runners/push.py

```diff
@@ -211,6 +211,8 @@ def organize(self, filename):
         common = {}
         with open(filename, "r") as policy:
             for line in policy:
+                # strip comments from the end of the line
+                line = re.sub(' #.*$', '', line)
```

Comment:
I know this has already been merged (sorry), but is the leading space in that RE going to be a problem? Like, if someone has a tab character before the comment, or (improbably) starts a comment with no whitespace between the policy line and the comment?